src/Wirtschaft/ABWL.tex

```diff
@@ -4,6 +4,372 @@
+                        
+                        Wenn eine lineare Abschreibung angewendet wird, können jedes Jahr 120,000 abgeschrieben werden. Da Abschreibungen den Gewinn reduzieren, muss das Unternehmen weniger Steuern zahlen.
+                        
+                \paragraph{Rentabilitätsrechnung}
+                    Bei der Rentabilitätsrechnung wird die Rentabilität verglichen.
+                    
+                \paragraph{Annuitätenrechnung}
+                    Bei der Annuitätenrechnung kumuliert man die Gewinne der Jahre und ermittelt so das Jahr, in dem sich die Anlage annulliert hat.
+                    
+            \subsubsection{Dynamische Verfahren}
+                \paragraph{Kapitalwertmethode}
+                    \begin{equation}
+                        C_0 = \left(e - a\right)\frac{\left(1 + i\right)^n - 1}{i \cdot \left( 1 + i \right)^n} + R\cdot \left( 1 + i \right)^{-n} - A_0
+                    \end{equation}
+                    
+                    \begin{itemize}
+                        \item[$e$] Einzahlungen/Einnahmen
+                        \item[$a$] Auszahlungen/Ausgaben
+                        \item[$i$] Renditeerwartungen des Managements
+                        \item[$n$] Nutzungsdauer
+                        \item[$R$] Restwert
+                        \item[$A_0$] Anschaffungsauszahlungen
+                    \end{itemize}
+                    
+                    \subparagraph{Gewinn}
+                        Der Gewinn wird mit $\left(e - a\right)$ in der Formel ausgedrückt.
+                    
+                    \subparagraph{Abzinsfaktor}
+                        Der Abzinsfaktor (AbF) $\left( 1 + i \right)^{-n}$ wird verwendet, um einen Betrag über $n$ Jahre abzuzinsen. Das Bedeutet, wenn man nach $n$ Jahren einen Restbetrag von $x$ hat, hätte man auch $x$ für $n$ Jahre bei einem Zinssatz von $i$ anlegen können.
+                        
+                    \subparagraph{Aufzinsfaktor}
+                        Der Aufzinsfaktor (AuF) $\left( 1 + i \right)^{n} = \frac{1}{\text{AbF}}$ wird verwendet, um einen Betrag über $n$ Jahre aufzuzinsen.
+                        
+                    \subparagraph{Diskontierungsfaktor}
+                        Der Diskontierungsfaktor (DSF) $\frac{\left(1 + i\right)^n - 1}{i \cdot \left( 1 + i \right)^n}$ wird verwendet, um einen jährlichen Gewinn über $n$ Jahre abzuzinsen.
+                        
+                    \subparagraph{Kapitalwiedergewinnungsfaktor}
+                        Der Kapitalwiedergewinnungsfaktor (KwF) ist das Reziprok des Diskontierungsfaktor.
+                        
+                    \subparagraph{Bedeutung von $C_0$}
+                        Wenn man mit der Kapitalwertmethode $C_0 = 0$ berechnet, erfüllt das Vorhaben die Renditeerwartung $i$ genau. Bei Werten über $0$ hat man eine höhere Rendite. Bei Werten kleiner als $0$ muss an den Parametern arbeiten oder das Vorhaben aufgeben.
```

Comment:
Should be: `muss an den Parametern gearbeitet oder das Vorhaben aufgegeben werden`